test/wrapper_array.jl

```diff
@@ -52,9 +54,24 @@ end
         @test nda_mw[c=2] == ones(10,20)
         @test names(nda_mw[c=2]) == (:_, :_)
     end
+
+    @testset "newaxis" begin
+        newaxis = [CartesianIndex()];
+
+        @test names(nda[:,newaxis,:]) == (:x, :_, :y)
+        @test size(nda[:,newaxis,:]) == (2, 1, 2)
+
+        @test names(nda[1,newaxis,1]) == (:_,)
+        @test size(nda[1,newaxis,1]) == (1,)
+
+        @test names(nda[CartesianIndex(1,1),newaxis]) == (:_,)
+        @test size(nda[CartesianIndex(1,1),newaxis]) == (1,)
+        @test nda[CartesianIndex(1,1),newaxis][1] == nda[1,1]
```

Comment:
minor style point: would be good to stick to the Blue style of spaces after commas throughout this block